Directory.Packages.props

```diff
@@ -6,41 +6,47 @@
-		<PackageVersion Include="Microsoft.AspNetCore.Components.WebAssembly.Authentication" Version="8.0.3" Condition="'$(TargetFramework)' == 'net8.0'" />
-		<PackageVersion Include="Microsoft.AspNetCore.Components.WebAssembly.Authentication" Version="7.0.17" Condition="'$(TargetFramework)' == 'net7.0'" />
-		<PackageVersion Include="Microsoft.AspNetCore.Components.WebAssembly.Authentication" Version="6.0.28" Condition="'$(TargetFramework)' == 'net6.0'" />
-		<PackageVersion Include="Microsoft.AspNetCore.Components.WebAssembly" Version="8.0.3" />
-		<PackageVersion Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="8.0.3" />
-		<PackageVersion Include="Microsoft.AspNetCore.Components.WebAssembly.Server" Version="8.0.3" />
+		<PackageVersion Include="LoxSmoke.DocXml" Version="3.6.1" />
+		<PackageVersion Include="Microsoft.AspNetCore.Components.Web" Version="8.0.4" Condition="'$(TargetFramework)' == 'net8.0'" />
+		<PackageVersion Include="Microsoft.AspNetCore.Components.Web" Version="7.0.18" Condition="'$(TargetFramework)' == 'net7.0'" />
+		<PackageVersion Include="Microsoft.AspNetCore.Components.Web" Version="6.0.29" Condition="'$(TargetFramework)' == 'net6.0'" />
+		<PackageVersion Include="Microsoft.AspNetCore.Components.WebAssembly.Authentication" Version="8.0.4" Condition="'$(TargetFramework)' == 'net8.0'" />
+		<PackageVersion Include="Microsoft.AspNetCore.Components.WebAssembly.Authentication" Version="7.0.18" Condition="'$(TargetFramework)' == 'net7.0'" />
+		<PackageVersion Include="Microsoft.AspNetCore.Components.WebAssembly.Authentication" Version="6.0.29" Condition="'$(TargetFramework)' == 'net6.0'" />
+		<PackageVersion Include="Microsoft.AspNetCore.Components.WebAssembly" Version="8.0.4" Condition="'$(TargetFramework)' == 'net8.0'" />
+		<PackageVersion Include="Microsoft.AspNetCore.Components.WebAssembly" Version="7.0.18" Condition="'$(TargetFramework)' == 'net7.0'" />
+		<PackageVersion Include="Microsoft.AspNetCore.Components.WebAssembly" Version="6.0.29" Condition="'$(TargetFramework)' == 'net6.0'" />
+		<PackageVersion Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="8.0.4" Condition="'$(TargetFramework)' == 'net8.0'" />
+		<PackageVersion Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="7.0.18" Condition="'$(TargetFramework)' == 'net7.0'" />
+		<PackageVersion Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="6.0.29" Condition="'$(TargetFramework)' == 'net6.0'" />
+		<PackageVersion Include="Microsoft.AspNetCore.Components.WebAssembly.Server" Version="8.0.4" Condition="'$(TargetFramework)' == 'net8.0'" />
+		<PackageVersion Include="Microsoft.AspNetCore.Components.WebAssembly.Server" Version="7.0.18" Condition="'$(TargetFramework)' == 'net7.0'" />
+		<PackageVersion Include="Microsoft.AspNetCore.Components.WebAssembly.Server" Version="6.0.29" Condition="'$(TargetFramework)' == 'net6.0'" />
```

Comment:
@EnchantedCoder What type of NuGet package restore would failure did you encounter?
These packages (3) are only used in the documentation, which is `net8.0`-only and does not use multi-targeting.

Reply:
Havit.Blazor.Grpc.Client was failing to restore packages, complaining that the 8.0.x version is only compatible with .NET 8, as soon as I updated to 8.0.4. Making that change in the central package management file allowed it to restore and build normally again.

Reply:
Interesting, I will try to replay that scenario on my side.
(`Havit.Blazor.Grpc.Client` does not reference any of the `Microsoft.AspNetCore.Components.WebAssembly.*` packages.)

Reply:
Merged into master, tried to remove the .NET6 + 7 PackageVersions for those three packages and still able to restore (incl. `dotnet restore -f`).

@EnchantedCoder I will push that commit to master. Can you please try to double-check on your side?